packages/arb-token-bridge-ui/scripts/tsconfig.json

```diff
@@ -15,5 +15,6 @@
   },
   "compilerOptions": {
     // typescript options here
+    "moduleResolution": "NodeNext"
```

Comment:
NIT: Any related reason why we introduced this in the PR?

Reply:
the newer version of typescript throws an error which causes `yarn build` to fail if `moduleResolution` is not explicitly set to `NodeNext` 

```bash
/Users/fionna/offchain/arbitrum-token-bridge/packages/arb-token-bridge-ui/node_modules/ts-node/src/index.ts:859
    return new TSError(diagnosticText, diagnosticCodes, diagnostics);
           ^
TSError: ⨯ Unable to compile TypeScript:
error TS5109: Option 'moduleResolution' must be set to 'NodeNext' (or left unspecified) when option 'module' is set to 'NodeNext'.

    at createTSError (/Users/fionna/offchain/arbitrum-token-bridge/packages/arb-token-bridge-ui/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/Users/fionna/offchain/arbitrum-token-bridge/packages/arb-token-bridge-ui/node_modules/ts-node/src/index.ts:863:19)
    at /Users/fionna/offchain/arbitrum-token-bridge/packages/arb-token-bridge-ui/node_modules/ts-node/src/index.ts:1379:34
    at Object.compile (/Users/fionna/offchain/arbitrum-token-bridge/packages/arb-token-bridge-ui/node_modules/ts-node/src/index.ts:1451:13)
    at Module.m._compile (/Users/fionna/offchain/arbitrum-token-bridge/packages/arb-token-bridge-ui/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1219:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/fionna/offchain/arbitrum-token-bridge/packages/arb-token-bridge-ui/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1043:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  diagnosticCodes: [ 5109 ]
}
```